components/giftbox.vue

```diff
@@ -240,12 +246,20 @@
       }
 
       .btn-call-to-action {
-          display: inline-block;
-          overflow: hidden;
-          white-space: nowrap;
-          text-overflow: ellipsis;
-          width: fit-content;
-          max-width: 100%;
+        display: inline-block;
```

Comment:
These indentation fixes should automatically happen using [prettier](https://prettier.io/).
Best to check with @brampauwelyn how to configure that in VS Code.